external_services/exceptions.py

```diff
@@ -0,0 +1,10 @@
+class NotificationError(Exception):
+    """Exception raises an an error
+
+    Attributes:
+        message -- explanation of the error
+    """
+
+    def __init__(self, message="Sorry, there was a problem please try later"):
```

Comment:
All for more accurate errors :) should we add 'problem to the notification service' here?

Reply:
Error updated!!